### PR TITLE
Added Mattermore.io to the More Resources category, modified Category name.

### PR DESCRIPTION
**Pull request by iam-Shashank**
Mattermore.io sends weekly lists of companies working on solving climate change using Data Science/ AI. Also, sends alerts when opportunities open up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,11 @@ The format for companies is the following:
 Short description.
 ```
 
-### Agriculture & Food
+### Vegetation, Agriculture & Food
```
Comment (nglgzz) [inline README.md:30]: Can you change the title and link on the table of contents as well? It should be:

```
[Vegetation, Agriculture & Food](#vegetation-agriculture--food)
```

File: README.md [hunk 2]
```diff
@@ -27,7 +27,11 @@ The format for companies is the following:
...
+- [Droneseed](https://www.droneseed.co/) | Seattle, USA | 2015
+They plant native trees & vegetation using drone swarms and spray to protect them. Provide valuable insights to customers by collecting data in the field. Work with Governments, nonprofits, and private land owners as a service provider.
+- [Land Life Company.](https://landlifecompany.com/) | Amsterdam | 2013
```
Comment (nglgzz) [inline README.md]: This entry should be after Impossible Foods.

File: README.md [hunk 3]
```diff
@@ -165,3 +170,8 @@ Short description.
 ## License
 
 This work and all contributions to it are released into the public domain under the terms of the [CC0 1.0](./LICENSE).
+<!--stackedit_data:
```
Comment (nglgzz) [inline README.md]: I'm not sure what this is,but I don't think it's meant to be here :smile:

**Review by nglgzz — CHANGES_REQUESTED** [review]
Hey @iam-Shashank,  thank you for your awesome suggestions! 

I suggested some changes, they're mostly formatting issues. Also if you could add an empty line before and after a company name it would be great. 

Like this:
```

- [Name](url) | Location | Year

Description
```

Instead of:
```
- [Name](url) | Location | Year
Description
```

**Comment by iam-Shashank** [comment]
Will make changes. Sorry about the last few mistakes, I didn't realize that the 2nd set of changes were also in the pull request. The last mistake <!--stackedit_data:  is something which has come from https://stackedit.io which I was using for editing the README. Thanks for being so patient.😊

File: README.md (hunk 3 above)
Comment (iam-Shashank) [reply README.md]: Had to delink this repo from Stackedit, even after I deleted this line from the Readme on Github itself, it was auto-syncing and adding it again and again. Sorry about that. Now, it should be correct.

**Comment by iam-Shashank** [comment]
Have made the changes. Also, in one of the entries, although Markdown output was fine, there were extra spaces, which I have removed ( no changes in the final readable output) . I believe it was the Too Good To Go entry.

**Comment by nglgzz** [comment]
Great! Thank you for your contribution :slightly_smiling_face:

**Comment by iam-Shashank** [comment]
You are most welcome 😊

**Pull request merged**